test/test_autograd.py

```diff
@@ -855,6 +855,20 @@ def hook(grad):
             self.assertEqual(x.grad, expected_grad)
             self.assertIsNone(x_list[i].grad)
 
+    def test_grad_zero_grad_unused(self):
+        x = torch.tensor(0.5, dtype=torch.float32, requires_grad=True)
+        a = torch.tensor(1, dtype=torch.float32, requires_grad=True)
+        y = x * a
+        dydx = torch.autograd.grad(y, x, create_graph=True, allow_unused=True, zero_grad_unused=True)
+        d2ydx2_None = torch.autograd.grad(dydx, x, create_graph=True, allow_unused=True)
```

Comment:
nit: d2ydx2_None -> d2ydx2_none (lower case for variables)

Reply:
Renamed